client/components/thank-you/index.tsx

```diff
@@ -128,17 +148,23 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 
 	return (
 		<ThankYouContainer className={ classNames( 'thank-you__container', containerClassName ) }>
-			<ThankYouHeader className="thank-you__container-header">
+			<ThankYouHeader className={ classNames( 'thank-you__container-header', headerClassName ) }>
 				{ /* eslint-disable-next-line jsx-a11y/alt-text */ }
 				<img { ...thankYouImage } />
+				<ThankYouFooterContainer>
+					<h1> { thankYouImageFooter } </h1>
+					<h2> { thankYouImageFooterSubtitle } </h2>
+				</ThankYouFooterContainer>
```

Comment:
I really don't like these names. :-/ I was really confused because I was wondering where you'd added a footer -- I wasn't seeing anything at the bottom of the page.

I think it would be simpler if we refactored the logic here to remove the `ThankYouHeaderTitle` below, and call this `ThankYouTitleContainer`, and then renamed the props to `thankYouTitle` and `thankYouSubtitle`.
```suggestion
				<ThankYouTitleContainer>
					<h1>{ thankYouTitle }</h1>
					{ thankYouSubtitle && <h2>{ thankYouSubtitle }</h2> }
				</ThankYouTitleContainer>
```